src/sos-packaging.props

```diff
@@ -1,5 +1,4 @@
-<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
Don't we need a license headers for this file?

Reply:
Not on MSBuild files, and we are MIT, not apache. I'll remove the apache headers in a subsequent PR.